packages/components/src/__actions__/LinkButton/v3/_docs/LinkButton.doc.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { VisuallyHidden } from "~components/VisuallyHidden"
+import { Icon } from "~components/__future__"
+import { ReversedColors } from "~components/__utilities__/v3"
+import { LinkButton } from "../index"
+
+const meta = {
+  title: "Actions/LinkButton/LinkButton (v3)",
+  component: LinkButton,
+  args: {
+    children: "Label",
+    href: "#link-button-clicked",
+  },
+  argTypes: {
+    icon: {
+      options: ["delete", "arrow", "plus"],
+      mapping: {
+        delete: <Icon isPresentational name="delete" />,
+        arrow: <Icon isPresentational name="arrow_forward" />,
+        add: <Icon isPresentational name="add" />,
+      },
+    },
+  },
+} satisfies Meta<typeof LinkButton>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Playground: Story = {}
+
+export const LinkButtonOpensInNewTab: Story = {
+  args: {
+    children: (
+      <>
+        Label
+        <VisuallyHidden> opens a new tab</VisuallyHidden>
+      </>
+    ),
+    href: "https://www.google.com",
+    target: "_blank",
+    icon: <Icon isPresentational name="open_in_new" shouldMirrorInRTL />,
+    iconPosition: "end",
+  },
+}
+
+export const LinkButtonVariants: Story = {
+  render: args => (
+    <>
+      <LinkButton {...args} variant="primary" />
+      <LinkButton {...args} variant="secondary" />
+      <LinkButton {...args} variant="tertiary" />
```

Comment:
It actually might be nice to change the label for these examples so that you have it say the variant name instead of Label. That way the reader can tell what is what without needing to open the code snippet (same with the reversed and sizes).